addons/stock/models/stock_move.py

```diff
@@ -881,6 +881,17 @@ def split_lots(self, lots):
     def action_generate_lot_line_vals(self, context, mode, first_lot, count, lot_text):
         assert mode in ('serial', 'import')
         default_vals = {}
+
+        def generate_lot_qty(quantity, qty_per_lot):
+            if qty_per_lot <= 0:
+                raise UserError(_("The quantity per lot should always be a positive value."))
+            line_count = int(quantity // qty_per_lot)
+            leftover = quantity % qty_per_lot
+            qty_array = [qty_per_lot for _ in range(line_count)]
```

Comment:
There is always something cool to learn with python :nerd_face: 
```suggestion
            qty_array = [qty_per_lot] * line_count
```